modules/investigating-etcd-installation-issues.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// *support/troubleshooting-installations.adoc
+
+[id="investigating-etcd-installation-issues_{context}"]
+= Investigating etcd installation issues
+
+If you experience etcd issues during installation, check etcd pod status and collect etcd pod logs. Also, verify etcd DNS records and check DNS availability on master nodes.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* The `ssh-agent` process is active on your computer, and you provided both the `ssh-agent` process and the installation program the same SSH key.
+* If you tried to install a cluster on infrastructure that you provisioned, you must have the fully-qualified domain names of the control plane, or master, machines.
+
+.Procedure
+
+. Check the status of etcd pods:
+.. Review the status of pods in the `openshift-etcd` namespace:
++
+----
+$ oc get pods -n openshift-etcd
+----
++
+.. Review the status of pods in the `openshift-etcd-operator` namespace:
++
+----
+oc get pods -n openshift-etcd-operator
+----
+
+. If any of the pods listed by the previous commands are not showing `Running` or `Completed` status, gather diagnostic information for the pod:
+.. Review events for the pod:
++
+----
+$ oc describe pod/<pod_name> -n <namespace>
+----
++
+.. Inspect the pod's logs:
++
+----
+$ oc logs pod/<pod_name> -n <namespace>
+----
++
+.. If the pod has more than one container, the preceding command will error and the container names will be provided in the error message. Inspect logs for each container:
++
+----
+$ oc logs pod/<pod_name> -c <container_name> -n <namespace>
+----
+
+. If the API is not functional, review etcd pod and container logs on each master node by using SSH instead. Replace `<master-node>.<cluster_name>.<sub_domain>.<domain>` with appropriate values:
+.. List etcd pods on each master node:
++
+----
+$ ssh core@<master-node>.<cluster_name>.<sub_domain>.<domain> sudo crictl pods --name=etcd-member
+----
++
+.. For any pods not showing `Ready` status, inspect pod status in detail. Replace `<pod_id>` with the pod's ID listed in the output of the preceding command:
```

Comment:
@bobfuru thanks for your suggestion. My view is that if we added some sample outputs here, we should really add some throughout the PR. Is there a particular reason why you felt this might be useful in context to this one particular module?